addons/project/views/project_project_views.xml

```diff
@@ -238,12 +238,12 @@
                     <field name="date_start" string="Planned Date" widget="daterange" options="{'end_date_field': 'date'}" required="False" optional="hide"/>
                     <field name="date" column_invisible="True" />
                     <field name="milestone_progress" widget="progressbar"
-                        invisible="milestone_progress == 0 or not allow_milestones"
+                        column_invisible="milestone_progress == 0"
```

Comment:
why do we no longer use `allow_milestones`?

Why do you use `column_invisible` instead of `invisible`?